gef.py

```diff
@@ -10829,9 +10925,29 @@ def version(self) -> Optional[Tuple[int, int]]:
         if not is_alive():
             return None
         if not self._version:
-            self._version = get_libc_version()
+            self._version = GefLibcManager.find_libc_version()
         return self._version
 
+    @staticmethod
+    def find_libc_version() -> Tuple[int, ...]:
```

Comment:
is this something we should cache?

Reply:
we probably should, but adding the "@lru_cache()" another makes the tests on Ubuntu 20.04 fail (the still pass on 22.04). I'm guessing that it has something to do with a `staticmethod` not working with the annotation in the python version in 20.04. I haven't investigated... Do you know something about this?

Reply:
that is strange. Have you tried swapping the order of the decorators?

Reply:
let's try...

Reply:
interesting, this seems to have worked now

Reply:
I think it's because we return a cached version, but that version is not the version that is turned into a static method, so calling it is not cached. This presumably was fixed in later versions of python. We could check, but I don't really care.